features/state/processor.feature

```diff
@@ -0,0 +1,32 @@
+Feature: State processor
+  In order to use the API independently of the framework
+  As a software developer
+  I need to be able to use state processor in API resources
```

Comment:
```suggestion
  I need to be able to set classes which aren't marked with #[ApiResource] as input and output
```

Reply:
done in commit eada6c0bc00762ec50a0c4a2b9889df3bcbf65ae